src/Model/IssueModel.php

```diff
@@ -42,7 +42,7 @@ class IssueModel
     private $state;
 
     /**
-     * @var array<int,string>
+     * @var array<int,array<string>>
```

Comment:
I think `array<int, array<string, mixed>` would be more correct here

Reply:
And just add `array<string>` as return annotation to the `getLabels` method. Should work then :)

Reply:
@icanhazstring I'm getting a syntax error when I try specifying the type for getLabels().
![image](https://user-images.githubusercontent.com/20955511/95002203-1186fa00-05da-11eb-836f-d0b26db8aab0.png)

Reply:
Php itself only allows return type to be of `array`. You need to add an annotation to that.

```
/*
 * @return array<string>
 */
```

Reply:
Thanks! Should be good now.